controllers/comments-controllers.js

```diff
@@ -16,3 +19,20 @@ exports.getCommentsByArticleId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postCommentByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const { body } = req;
+  const proms = [
+    checkExists("articles", "article_id", article_id),
+    insertCommentByArticleId(body, article_id),
```

Comment:
im not a big fan of passing the body right in but its not something that must be changed, I think its fine as it is but we want to separate the request from the model and if I wanted to change the keys the body was sending, my first instinct would be to check the controller :)